### PR TITLE
Add support for Microsoft Lync

### PR DESCRIPTION
**Pull request by amatos**
added mackup/applications/microsoft-lync.cfg, updated changelog.md and readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -255,6 +255,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
   - [Mercurial](http://mercurial.selenic.com/)
   - [MercuryMover](http://www.heliumfoot.com/mercurymover/)
   - [Messages](http://www.apple.com/osx/apps/#messages)
+  - [Microsoft Lync](http://office.microsoft.com/lync)
```
Comment (lra) [inline README.md]: The link is not working for me, it redirects me to some skype page =(

Comment (amatos) [inline README.md]: Ah… Microsoft is re-branding Lync as Skype for Business.  But, I didn’t realize that it wouldn’t provide a download link unless one’s microsoft account was tied to a business that used Lync.  I think this particular entry would be of extremely limited use, so it probably wouldn’t make sense to add it to the master.

On Apr 20, 2015, at 11:15 PM, Laurent Raufaste <notifications@github.com<mailto:notifications@github.com>> wrote:


In README.md<https://github.com/lra/mackup/pull/523#discussion_r28747339>:

> @@ -255,6 +255,7 @@ See the [README](doc/README.md) file in the doc directory for more info.
>    - [Mercurial](http://mercurial.selenic.com/)
>    - [MercuryMover](http://www.heliumfoot.com/mercurymover/)
>    - [Messages](http://www.apple.com/osx/apps/#messages)
> +  - [Microsoft Lync](http://office.microsoft.com/lync)


The link is not working for me, it redirects me to some skype page =(

**Comment by lra** [comment]
So what do you suggest?
Also please keep it to 1PR per application, it's hard to keep track of things otherwise

**Comment by amatos** [comment]
I'm going to close the PR.  Between the limited use and soon-to-change name, I don't think this is of value.  Sorry about the add of Shimo.  I hadn't  realized it would append that change to this PR as well. I'm still relatively new to git and github.

**Comment by lra** [comment]
no worries ;)

**Pull request closed without merging**